src/components/Media/Iframe/Iframe.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../../styles/variables.scss';
+@import '../../../../styles/mixins';
+
+$block: '.#{$ns}media-component-iframe';
+
+#{$block} {
+    overflow: hidden;
+
+    &_margins {
+        margin: $indentSM;
+
+        #{$block}__item {
+            border-radius: 0;
+        }
+    }
+
+    #{$block}__iframe {
+        border-radius: 0;
+    }
+
+    &__item {
+        width: calc(100% + 4px);
```

Comment:
could you please use variables describing why there are 4px indents and -2px margin?